build/drand.go

```diff
@@ -32,9 +32,10 @@ const (
 var DrandConfigs = map[DrandEnum]dtypes.DrandConfig{
 	DrandMainnet: {
 		Servers: []string{
-			"https://api.drand.sh",
-			"https://api2.drand.sh",
-			"https://api3.drand.sh",
+			"http://api.drand.sh",
```

Comment:
I do not see any potential negative impact of doing this, unless GFW would prefer to block HTTP or delay traffic to cloudfront (?). If there's a concern, I can leave things as they are.

Reply:
I don't see any advantage to using http. If we can use https, we should use it